projects/js-packages/components/components/text/style.module.scss

```diff
@@ -70,14 +70,45 @@
 	line-height: 16px;
 }
 
-/* Headings */
-.headline-medium,
-.headline-small {
-	margin-bottom: calc( var(--spacing-base ) * 3 );
-}
+/** Box-Model: Spacing (margin) */
+$props: (
+	"margin": "m",
+	"padding": "p",
+);
 
-/* Title size variations: medium | small */
-.title-medium,
-.title-small {
-	margin-bottom: var( --spacing-base );
+$sides: (
+	"top": "t",
+	"right": "r",
+	"bottom": "b",
+	"left": "l",
+);
+
+@each $prop, $p in $props {
+	$sizes: ( 0, 1, 2, 3, 4, 5, 6, 7, 8 );
```

Comment:
This put a limit on us, that's the reason I vote for us to move for some `css-in-js` solution that helps with this integration between CSS and JS, but let's keep this for the future.

Reply:
agree, the style and js are not connect which means we cannot share data between them.

Reply:
yes, at my P2: p9dueE-3Tq-p2 I highlighted this but waited to get more use cases to make this movement clearer.